### DISCUSSION TITLE
Live view from Home Assistant notification

### DISCUSSION THREAD

**Discussion by rajil** [blakeblackshear/frigate#15425] [Ask A Question]
Hello,
 I am using [this](https://gist.github.com/TheRealFalseReality/eab315e84f2711783fb7454ac0b42187) [blueprint](https://community.home-assistant.io/t/camera-send-save-snapshot-to-mobile-device-when-motion-is-detected-with-conditions-now-with-custom-triggers/604156) to send a notification to the phone with a camera snapshot when a bell button is pushed (binary sensor). The blueprint allows one to set a 'URL to Navigate' when the snapshot is clicked. I would like to set this to the liveview of the camera.
How can i achieve this?

Thanks

PS: The camera is not exposed externally however HA can access it.

**Comment by hawkeye217** [comment]
`http://frigate_ip:port/#cameraname`

**Comment by hawkeye217** [comment]
If you can't access your frigate instance externally, you'll need to set up a dashboard in HA with the camera on it that you can access with a URL, which you can set in the dashboard configuration dialog in HA.

**Comment by rajil** [comment]
Thanks. My frigate is exposed externally, so that helped.

Is it possible to go to specific time in the recorded stream? The camera is recording 24x7.

**Reply by NickM-27** [reply]
You can jump to the recordings of a specific review item

**Reply by hawkeye217** [reply]
No, there's no url available for that specifically. However, you can navigate to a review item's footage in History by `frigate_ip:port/review?=id=<review_item_id>`. Review item IDs can be seen through the /reviews MQTT topic.
